ts/features/payments/checkout/saga/networking/__tests__/handleWalletPaymentGetUserWallets.test.ts

```diff
@@ -38,7 +38,7 @@ describe("Test handleWalletPaymentGetUserWallets saga", () => {
     testSaga(
       handleWalletPaymentGetUserWallets,
       mockGetWalletsByIdUser,
-      paymentsGetPaymentUserMethodsAction.request()
+      paymentsGetPaymentUserMethodsAction.request({})
```

Comment:
You need to add a test to verify that the onResponse callback is called.

Reply:
Addressed into 84cce8af1e4d661c14fdbe49b7e312c741e92e54